public/javascripts/main.js

```diff
@@ -2,4 +2,8 @@
 const hello = "hello world";
 console.log(hello);
 
-if(hello) console.log('hi');
\ No newline at end of file
+if(hello) console.log('hi');
+
+function homebFunc() {
+  window.location.href="";
```

Comment:
Let's direct this to the topics.html page

Reply:
Why don't we remove "browse topics" from the list and add this button that redirects to the topics.html page

Reply:
Sounds like a good idea

Reply:
oh i think i understand. the list is supposed to be the navbar. So that would just go at the top. Our index page is supposed to be a "landing page" and in every landing page there's a hero section with a call to action button. That's what the "Start Chatting" button is for. So it really doesn't hurt to have both.

Reply:
Just do whatever makes your life easier lol